### ISSUE TITLE
failed to resolve: use of undeclared type or module `exitcode`

### ISSUE THREAD

**Issue by stappersg** [rust-lang-nursery/cli-wg#137]
Hi,

In-depth topic 2.3 *Exit codes* has in its example lines like these:
```
        Ok(_) => {
            println!("Done!");
            std::process::exit(exitcode::OK);
        }
```

Upon compiling simular lines I do get
```
error[E0433]: failed to resolve: use of undeclared type or module `exitcode`
  --> src/main.rs:54:28
   |
54 |         std::process::exit(exitcode::OK);
   |                            ^^^^^^^^ use of undeclared type or module `exitcode`

```

I think the example code needs something like
```
use std::process::exit;
```
but adding such line is not enough.

**Comment by killercup** [comment]
Did you add exitcode as a dependency?

**Comment by Dylan-DPC** [comment]
There is no mention of the dependency in the book anywhere. If you can make a PR that adds a note about the dependency used, it will be helpful. Thanks

**Comment by stappersg** [comment]
> If you can make a PR that adds a note about the dependency used, it will be helpful.

There is now merge request #138 

> Thanks

:-)

**Comment by stappersg** [comment]
Heart beat request

**Comment by killercup** [comment]
Fixed by #138
